Libraries/NativeModules/specs/NativeDialogManagerAndroid.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+import Platform from '../../Utilities/Platform';
+
+type DialogAction = 'buttonClicked' | 'dismissed';
+type DialogButtonKey = 'buttonPositive' | 'buttonNegative' | 'buttonNeutral';
```

Comment:
@RSNara, I saw one of your comments in another PR about the codegen not supporting enum or union types. So, based on that, these two types (`DialogAction` and `DialogButtonKey`) are not supported by the codegen, right?